### PR TITLE
Add command to deploy zkevm on sequencer

### PR DESCRIPTION
**Pull request by nomaxg**

### CODE REVIEW

**Review by jbearer — APPROVED** [review]

File: justfile [hunk 1]
```diff
@@ -78,5 +78,9 @@ build-docker-l1-geth:
 
 build-docker: build-docker-l1-geth build-docker-zkevm-node build-docker-zkevm-adaptor
 
+deploy-on-sequencer:
```
Comment (sveitser) [inline justfile:81]: @nomaxg This is the same as the `just deploy-rollup --force-recreate --renew-anon-volumes`, on line 17.

**Review by sveitser — CHANGES_REQUESTED** [review]
I think we can consolidate the justfile commands to avoid some duplication.

File: justfile (hunk 1 above)
Comment (nomaxg) [reply justfile:81]: done

Comment (sveitser) [reply justfile:81]: @nomaxg `deploy-rollup` already calls the `deploy-contracts` just command.

```
deploy-rollup *args: deploy-contracts
docker compose --profile zkevm1 --profile zkevm1-preconfirmations --env-file .env --env-file deployment.env up {{args}}
```

It's easy to overlook, after the colon.

Comment (nomaxg) [reply justfile:81]: I see! So I guess this command isn't needed at all?

**Pull request closed without merging**